### PR TITLE
JSON Logger & web status page

### PR DESCRIPTION
**Pull request by rnevet**
Created a new logging option,  log to json file:
- status can be completely seen (including all coins)
- log size can be controlled
- can be parsed like in the lendingbot.html example

### CODE REVIEW

**Comment by Mikadily** [comment]
Good job as always.
But I'm thinking how could we improve. It's a bit complicated to start using. How difficult is to start a webserver automatically on python to serve that html? So that it would be easy to use on headless server.

**Comment by Mikadily** [comment]
And sorry for slow response.

**Comment by rnevet** [comment]
I fixed a bug, it should load the botlog.json file automatically if it is in the same folder as the lendingbot.html and the browser allows it. (Internet explorer gives a warning that can be ignored, Chrome doesn't allow auto loading local files)

It is possible to start a local server using the following command (from the www folder):
'''
python -m SimpleHTTPServer 8000
'''
then the page will be available under:
http://127.0.0.1:8000/lendingbot.html

This is really dependent on the personal setup, i.e. if you are anyway running a webserver - you might not want to run another one.

**Comment by rnevet** [comment]
I'll add an option to start a web server in the www folder.

**Comment by Mikadily** [comment]
Awesome.

Now, I'm not so experienced with git. Maybe you know, how should I merge this update as a single commit? Or shouldn't I? It looks a bit ugly at the moment :)

**Comment by rnevet** [comment]
Added WebServer option and squashed the commits locally, forced pushed to my repo - make it all nice looking.

File: lendingbot.py [hunk 1]
```diff
@@ -231,32 +248,75 @@ def setAutoRenew(auto):
...
+			# quiet server logs
+			def log_message(self, format, *args):
+				return
+			# serve from www folder under current working dir
+			def translate_path(self, path):
+				return SimpleHTTPServer.SimpleHTTPRequestHandler.translate_path(self, '/www' + path)
+		
+		global server
+		server = SocketServer.TCPServer((HOST, PORT), QuietHandler)
+		print 'Started WebServer, lendingbot status available at http://'+ HOST +'/' + str(PORT) + '/lendingbot.html'
```
Comment (Mikadily) [inline lendingbot.py]: There's a mistake, ':' after HOST

**Comment by Mikadily** [comment]
There's a mistake at the startup screen:

> Started WebServer, lendingbot status available at http://127.0.0.1/8000/lendingbot.html

And also the log file is not in the same dir as the html.

Otherwise it looks great, so impatient to merge it :)

Again sorry for me being slow.

**Comment by rnevet** [comment]
Fixed the mistake in the startup message.
Make sure that the jsonfile config should be: www\botlog.json  - it wasn't in the first commit.

**Comment by Mikadily** [comment]
Awesome. Happy with the result.

**Pull request merged**